src/main/frontend/src/components/itempage/ItemDetails.js

```diff
@@ -0,0 +1,62 @@
+import "./ItemDetails.css";
+import { ITEM_PAGE_TAB_VALUES } from "../../utils/constants";
+import { useState } from "react";
+
+function ItemDetails({ details }) {
+  const [selectedTab, setSelectedTab] = useState(
+    ITEM_PAGE_TAB_VALUES["DETAILS"]
+  );
+
+  const renderButton = (title, value) => {
+    if (
+      value === ITEM_PAGE_TAB_VALUES["SELER_INFO"] ||
+      value === ITEM_PAGE_TAB_VALUES["CUSTOMER_REVIEWS"]
+    ) {
+      return (
+        <button
+          disabled
+          className={
+            selectedTab === value
+              ? "item_option_button selected_item_option_button"
+              : "item_option_button"
+          }
+          onClick={() => {}}
```

Comment:
With empty onClick functions those are still not functional buttons. Add currentTab state and change it on buttons click. Then render tab values based on this state